R/geom-rug.r

```diff
@@ -4,17 +4,14 @@
 #' with the two 1d marginal distributions. Rug plots display individual
 #' cases so are best used with smaller datasets.
 #'
-#' The rug lines are drawn with a fixed size (3\% of the total plot size) so
-#' are dependent on the overall scale expansion in order not to overplot
-#' existing data.
```

Comment:
I think the deleted lines 7-9 contain important information and should be kept. But edit so it's clear that the default size can be modified.

Example:
> By default, the rug lines are drawn with a length that corresponds to 3\% of the total plot size. Since the default scale expansion of for continuous variables is 5\% at both ends of the scale, the rug will not overlap with any data points under the default settings.